chat-plugins/info.js

```diff
@@ -1344,6 +1344,7 @@ exports.commands = {
 		}
 
 		const isOMSearch = (cmd === 'om' || cmd === 'othermetas');
+		const isBanlistSearch = (cmd in ['viewruleset', 'viewbanlist', 'ruleset', 'banlist']);
```

Comment:
... I'm pretty sure this isn't a thing? ?_?

```Javascript
const banlistSearches = ['viewruleset', 'viewbanlist', 'ruleset', 'banlist'];
const isBanlistSearch = banlistSearches.includes(cmd);
```

Reply:
Oh yeah